src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct

```diff
@@ -144,10 +141,7 @@
 
       <Button guid="guidInteractiveWindowCmdSet" id="cmdidHistoryNext" priority="0x0300" type="Button">
         <Parent guid="guidInteractiveWindowCmdSet" id="ConsoleMenuGroup"/>
-        <Icon guid="ImageCatalogGuid" id="MoveDown" />
-        <!-- TODO (https://github.com/dotnet/roslyn/issues/5295)
         <Icon guid="ImageCatalogGuid" id="ListMembers" />
```

Comment:
Yes, `ListMembers` was an existing icon with an up arrow over some lines representing text and UX made us a corresponding down icon.